lib/blueprinter/view_collection.rb

```diff
@@ -89,5 +91,14 @@ def add_to_ordered_fields(ordered_fields, definition, fields, view_name_filter =
         ordered_fields[definition.name] = fields[definition.name]
       end
     end
+
+    def gather_transformers_from_included_views(view_name)
```

Comment:
**nit**:  Should we make this `private`?

Reply:
Isn't it already?